### ISSUE TITLE
Issue: Unhandled rejection M_UNKNOWN: Internal server error

### ISSUE THREAD

**Issue by RameshThota87** [matrix-org/matrix-appservice-slack#77]
Hi guys,
I installed Home server(HS) and Application server(AS) into two different machines.
I executed the following steps:
`node app.js -r -f slack-registration.yaml -u "http://localhost:9898"(Slack Appservice URL) -c config/config.yaml -l slackbot`

Copy the newly-generated slack-registration.yaml file to the homeserver. Add the registration file to your homeserver config (default homeserver.yaml):

```
app_service_config_files:
...
"/path/to/slack-registration.yaml"
```
Restarted the Home server

`node app.js -c config.yaml -p 8008(Home server port)`

```
Loading config file /../matrix-appservice-slack/config/config.yaml
Matrix-side listening on port 8008
Slack-side listening on port 9898 over http
Accepting pending invites
[-] GET http://matrix.XXXXXXX.com:8008/_matrix/client/r0/sync (AS) Body:
[-] GET http://matrix.XXXXXXX.com:8008/_matrix/client/r0/sync (AS) HTTP 500 Error: {"errcode":"M_UNKNOWN","error":"Internal server error"}
Unhandled rejection M_UNKNOWN: Internal server error
```
If I checked same thing in Postman client, I'm getting proper response. (http://matrix.XXXXXXX.com:8008/_matrix/client/r0/sync)

Can anyone help me, what could be the issue?

Thanks in advance

**Comment by Half-Shot** [comment]
This is because https://github.com/matrix-org/matrix-appservice-slack/blob/3de59d617f06e8de35ed6c62b80ab7ecda3604d6/lib/Main.js#L629 is no longer supported since we cannot sync to the homeserver as a appservice. The code should be removed.

**Comment by Half-Shot** [comment]
This was fixed in #80
